src/Forcings/advective_forcing.jl

```diff
@@ -45,7 +45,7 @@ settling = AdvectiveForcing(UpwindBiasedFifthOrder(), w=w_Stokes)
 AdvectiveForcing with the UpwindBiased scheme:
 ├── u: ZeroField{Int64}
 ├── v: ZeroField{Int64}
-└── w: ConstantField(-1.97096)
+└── w: ConstantField(-1.970964448157431)
```

Comment:
Is this because we aren't using `prettysummary` anymore? I think we should use it --- it could also help make things look cleaner if running with different precisions becomes common

Reply:
ok

Reply:
Looks like we still aren't using `prettysummary`